packages/ove-asset-manager/src/OVE.Service.AssetManager/Views/Home/Index.cshtml

```diff
@@ -6,6 +6,6 @@
     <div class="col-md-3" style="width:100%">
         <h2>Asset Manager Service</h2>
         <p>Please visit the <a asp-area="" asp-controller="OVEAssetModel" asp-action="Index">Asset List</a> to upload a new asset</p>
-        <p>This site has a full API which is auto-documented <a href="~/swagger/">here </a></p>
+        <p>This site has a full API which is auto-documented <a href="~/api-docs/">here </a></p>

```

Comment:
I don't see how this information if helpful for the user. I has to be part of the Readme and dev guide, but not on the UI. It is not related to the fact that the route changed from swagger to api-docs, but to the fact that this message should be removed entirely from the UI.

Reply:
ok I've removed the link though i don't think it does any harm ;)